api/nodes/views.py

```diff
@@ -258,9 +468,79 @@ def get_queryset(self):
 
 
 class NodeContributorDetail(generics.RetrieveUpdateDestroyAPIView, NodeMixin, UserMixin):
-    """Detail of a contributor for a node.
+    """Detail of a contributor for a node. *Writeable*.
+
+    Contributors are users who can make changes to the node or, in the case of private nodes,
```

Comment:
Again, mostly for sanity when making updates, I'm in favor of never having duplicated blocks of documentation.